test/src/main/java/org/apache/accumulo/test/VolumeChooserIT.java

```diff
@@ -299,19 +291,23 @@ public void twoTablesRandomVolumeChooser() throws Exception {
 
     // Create namespace
     try (AccumuloClient client = Accumulo.newClient().from(getClientProperties()).build()) {
-      createAndVerify(client, namespace1, v1 + "," + v2 + "," + v3);
-      createAndVerify(client, namespace2, v1 + "," + v2 + "," + v3);
+      createAndVerify(client, namespace1, v1 + "", v1 + "," + v2 + "," + v3);
+      createAndVerify(client, namespace2, v2 + "", v1 + "," + v2 + "," + v3);
     }
   }
 
-  private void createAndVerify(AccumuloClient client, String ns, String expectedVolumes)
-      throws Exception {
+  private void createAndVerify(AccumuloClient client, String ns, String preferred,
+      String expectedVolumes) throws Exception {
     client.namespaceOperations().create(ns);
 
     // Set properties on the namespace
     client.namespaceOperations().setProperty(ns, PERTABLE_CHOOSER_PROP,
         RandomVolumeChooser.class.getName());
 
+    // The random volume chooser should not use this property, so setting it should not cause a
+    // problem
+    client.namespaceOperations().setProperty(ns, PREFERRED_CHOOSER_PROP, preferred);
+
```

Comment:
I think the attempt to reuse this createAndVerify method for different test cases is very confusing. We should just get rid of the method, and do the work in each place to create the namespace with the desired config for each case.